lib/utils/__tests__/isContextFunctionalPseudoClass.test.js

```diff
@@ -4,45 +4,33 @@ const isContextFunctionalPseudoClass = require('../isContextFunctionalPseudoClas
 const parseSelector = require('postcss-selector-parser');
 const postcss = require('postcss');
 
-function selector(css, cb) {
+function pseudo(css) {
+	const pseudos = [];
+
 	postcss.parse(css).walkRules((rule) => {
 		parseSelector((selectorAST) => {
-			selectorAST.walkPseudos(cb);
+			selectorAST.walkPseudos((pseudo) => pseudos.push(pseudo));
```

Comment:
Parameter `pseudo` is the same as function name `pseudo`. Maybe rename function to `pseudoClass` to avoid potential hard to find bugs.

Reply:
Good catch! ESLint's [`no-shadow`](https://eslint.org/docs/rules/no-shadow) rule may reduce our review cost:

```console
$ npx eslint --rule "no-shadow: 2" lib/utils/__tests__/isContextFunctionalPseudoClass.test.js

/tmp/stylelint/lib/utils/__tests__/isContextFunctionalPseudoClass.test.js
  12:29  error  'pseudo' is already declared in the upper scope  no-shadow

✖ 1 problem (1 error, 0 warnings)
```